core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2025,6 +2025,15 @@ package object config {
       .checkValue(v => v >= 0, "The threshold should be non-negative.")
       .createWithDefault(1L * 1024 * 1024)
 
+    private[spark] val CLEAN_BROADCAST_AFTER_EXECUTION_ENABLED =
+      ConfigBuilder("spark.broadcast.clean_after_execution.enabled")
```

Comment:
Please, use the Camel case. Look at the other configs in the file.

Reply:
Thank you, done